tests/unit/test_assess_vital.py

```diff
@@ -0,0 +1,25 @@
+from analyzer.utils.assessments import assess_vital
+
+
+def test_assess_vital():
+    """
+    test the assess_vital module
+    """
+    all_connected = {
+        "0": set(["user0", "user1", "user2", "user3", "user4"]),
+    }
+    all_vital: dict[str, set[str]] = {
+        "0": set(),
+    }
+
+    # For testing purposes we're setting 1 for each parameters of vital
+    all_vital = assess_vital(
+        all_connected=all_connected,
+        w_i=1,
+        VITAL_T_THR=1,  # 1 time to be considered as vital
```

Comment:
It would be good to test this for parameter values that are not the last 1 of 1 but maybe the last 3 of 4 to also test whether the script correctly handles cases that are not X of X but X of Y